### PR TITLE
clarified heroku docs, fixes #42

### LINKED ISSUE #42: Test-drive the Heroku HOWTO
http://apostrophecms.org/docs/tutorials/howtos/deploying-apostrophe-in-the-cloud.html

Let me know what's good, what's bad, what's unclear, what's buggy... the target audience is anybody who's built a few Apostrophe sites, so don't feel it's your job to somehow get what I didn't say. Thanks!

### PR DESCRIPTION
**Pull request by marykatefain**
I went through the deploying with Heroku tutorial, and it pretty much worked fine! Just a couple of things I thought could be clarified.

### CODE REVIEW

File: docs/tutorials/howtos/deploying-apostrophe-in-the-cloud.md [hunk 1]
```diff
@@ -51,7 +51,9 @@ mongodb://<dbuser>:<dbpassword>@dsxxxxx.mlab.com:xxxxx/YOUR-database-name
 
 You'll need to create a database user. Click on the "Users" tab, then click "Add database user" and create a user.
 
-Now you can create the complete URI, which will have the database user and password you just created.
+> Be careful to not use special symbols like '@' or '/' in your password. They will break your URI.
```
Comment (boutell) [inline docs/tutorials/howtos/deploying-apostrophe-in-the-cloud.md]: I would add "unless escaped, which is awkward."

Comment (stuartromanek) [reply docs/tutorials/howtos/deploying-apostrophe-in-the-cloud.md]: Is there a more developer'y way to caveat this than giving two examples? Is there a name for the characters-needing-escaping?

Comment (boutell) [reply docs/tutorials/howtos/deploying-apostrophe-in-the-cloud.md]: We're talking about characters that require escaping in URLs. I suggest we just recommend sticking to alphanumeric characters.

Comment (stuartromanek) [reply docs/tutorials/howtos/deploying-apostrophe-in-the-cloud.md]: I reworded this to `Stick to alphanumeric characters or be careful to escape symbols when creating a user.` good to go?

Comment (boutell) [reply docs/tutorials/howtos/deploying-apostrophe-in-the-cloud.md]: Yeah

**Pull request merged**